servatrice/migrations/servatrice_0021_to_0022.sql

```diff
@@ -0,0 +1,16 @@
+
+CREATE TABLE IF NOT EXISTS `cockatrice_audit` (
+  `id` int(7) unsigned zerofill NOT NULL auto_increment,
+  `id_server` tinyint(3) NOT NULL,
+  `name` varchar(35) NOT NULL,
+  `ip_address` varchar(255) NOT NULL,
+  `clientid` varchar(15) NOT NULL,
+  `incidentDate` datetime NOT NULL default '0000-00-00 00:00:00',
+  `action` varchar(35) NOT NULL,
+  `results` ENUM('fail', 'success') NOT NULL DEFAULT 'fail',
+  `details` varchar(255) NOT NULL,
```

Comment:
Database normalization: this is going to make the table grow in size quite fast, consider adding an integer field and us a different code number for every "action" and "details" (eg username invalid = 1, username already exists = 2, etc..).